### PR TITLE
[SPVR-60][SPVR-15] implement exitCode

### PR DESCRIPTION
**Pull request by Aceralon**
rely on https://github.com/projecteru2/libyavirt/pull/29

### CODE REVIEW

**Comment by CMGS** [comment]
WIP ?

**Comment by Aceralon** [comment]
> WIP ?

ready.

File: engine/engine.go [hunk 1]
```diff
@@ -13,9 +13,9 @@ import (
 type API interface {
 	Info(ctx context.Context) (*enginetypes.Info, error)
 
-	Execute(ctx context.Context, target string, config *enginetypes.ExecConfig) (execID string, stdout, stderr io.ReadCloser, stdin io.WriteCloser, _ error)
-	ExecResize(ctx context.Context, execID string, height, width uint) (err error)
-	ExecExitCode(ctx context.Context, execID string) (int, error)
+	Execute(ctx context.Context, ID string, config *enginetypes.ExecConfig) (result string, stdout, stderr io.ReadCloser, stdin io.WriteCloser, err error)
```
Comment (anrs) [inline engine/engine.go:16]: is the ```result string``` a PID? or PID for virt, something else for other engines?

Comment (Aceralon) [reply engine/engine.go:16]: for docker it will be `execID`, for vm it will be `pid`,

**Comment by anrs** [comment]
LGTM, maybe @jschwinger23 can take a look

**Pull request merged**